x-pack/plugins/fleet/common/constants/index.ts

```diff
@@ -23,7 +23,7 @@ export * from './download_source';
 // setting in the future?
 export const SO_SEARCH_LIMIT = 10000;
 
-export const FLEET_SERVER_INDICES_VERSION = 1;
+export const FLEET_SERVER_INDICES_VERSION = 2;
```

Comment:
I am not sure we still need this, it a rest of a time where Kibana was creating the `.fleet-*` indices

Reply:
yes it didn't seem to be used, should I remove the array completely?